### PR TITLE
Giveaway Wrap-up

### PR DESCRIPTION
**Pull request by SavoySchuler**
If your name is on this list, I have been unable to contact you. Would you please email your shirt size and address to saschule@microsoft.com as soon as possible? 

@Knightmeister
@Poopooracoocoo
@dotMorten 
@gvhung

@mrlacey and @navarro_mael - I have an email and Twitter DM out to you, respectively.

### CODE REVIEW

**Comment by SavoySchuler** [comment]
If your name is on this list, I have been unable to contact you. Would you please email your shirt size and address to saschule@microsoft.com as soon as possible? 

@Knightmeister
@Poopooracoocoo
@dotMorten 
@gvhung

@mrlacey and @navarro_mael - I have an email and Twitter DM out to you, respectively.

File: README.md [hunk 1]
```diff
@@ -8,59 +8,6 @@ WinUI is the easiest way to build great [Fluent Design](https://docs.microsoft.c
...
-
-
-[@Poopooracoocoo](https://github.com/Poopooracoocoo)
-
-
-[@dotMorten](https://github.com/dotMorten)
-
-
-
 #### Winners, keep an eye out for an email coming next week. We’re excited to keep working with you and evolving WinUI as your premiere UI framework on Windows!
```
Comment (chingucoding) [inline README.md]: Shouldn't this line be removed too? 😅

Comment (SavoySchuler) [reply README.md]: Nice catch - fixed!

**Comment by mrlacey** [comment]
> @mrlacey and @navarro_mael - I have an email and Twitter DM out to you, respectively.

Have emailed again - Microsoft mail filters tend to look down on my emails so check your span folder ;)

**Comment by knightmeister** [comment]
> If your name is on this list, I have been unable to contact you.

I've also emailed you this morning (my time). Let me know if you don't get my details.

**Pull request merged**